.travis.yml

```diff
@@ -218,6 +218,17 @@ matrix:
         sources: ['ubuntu-toolchain-r-test']
         packages: ['g++-9', 'ninja-build']
 
+  - os: linux
+    compiler: gcc
+    env:
+      - COMPILER=g++-9
```

Comment:
Please also add similar tests for Clang.

Reply:
Fine, I will do it

Reply:
Why did you close the PR?